build/AzurePipelinesTemplates/WindowsAppSDK-RunHelixTests-Job.yml

```
@@ -105,6 +107,26 @@ jobs:
       downloadPath: $(testPayloadDir)
       itemPattern: 'BuildOutput/$(buildConfiguration)/$(buildPlatform)/**/*'
 
+  - ${{ if eq(parameters.BuildForHelix, 'true') }}:
+    # Discover the tests to run and generate the Helix Work Items.
+    - task: powershell@2
+      displayName: 'DynamicDependencies.Win32: Discover tests'
+      condition: ${{ parameters.conditionForDiscoverTests }}
+      inputs:
+        targetType: filePath
+        filePath: $(winUIHelixPipelineScripts)\GenerateHelixWorkItems.ps1
+        arguments: -TestFilePattern 'DynamicDependency_Test_Win32\*.dll' -TestBinaryDirectoryPath '$(testPayloadDir)' -OutputProjFile '$(helixWorkItemsDir)\DynamicDependencies-Tests-Win32.proj' -WorkItemPrefix 'DynamicDependencies.Win32.$(buildPlatform).$(buildConfiguration)' -TestNamePrefix 'DynamicDependencies.Win32.$(buildPlatform).$(buildConfiguration)'
```

The path here (and next one below) should look something like this:

```suggestion
        arguments: -TestFilePattern 'BuildOutput\$(buildConfiguration)\$(buildPlatform)\DynamicDependency_Test_Win32\*.dll' -TestBinaryDirectoryPath '$(testPayloadDir)' -OutputProjFile '$(helixWorkItemsDir)\DynamicDependencies-Tests-Win32.proj' -WorkItemPrefix 'DynamicDependencies.Win32.$(buildPlatform).$(buildConfiguration)' -TestNamePrefix 'DynamicDependencies.Win32.$(buildPlatform).$(buildConfiguration)'
```